web/src/pages/administration/programs/vendors/VendorProgramRequestPage.vue

```diff
@@ -25,13 +25,19 @@
                   }})</strong
                 >
               </p>
-              <p class="mb-2">By: <strong>Michael Johnson</strong> (michael@icefoganalytics.com)</p>
-
-              <p class="mb-5">
-                <a href="mailto:michael@icefoganalytics.com">
-                  <v-icon class="mr-2">mdi-email</v-icon>Send Michael Johnson an email</a
-                >
-              </p>
+              <div v-if="!isNil(requestedByUser)">
+                <p class="mb-2">
+                  By:
+                  <strong> {{ requestedByUser.displayName }}</strong> ({{ requestedByUser.email }})
+                </p>
+
+                <p class="mb-5">
+                  <a :href="'mailto:' + requestedByUser.email">
+                    <v-icon class="mr-2">mdi-email</v-icon>Send {{ requestedByUser.displayName }} an
```

Comment:
Consider using a format string here with a bare word.
e.g. `"mailto:${requestedByUserEmail}"` as this make the string computation simple.

I have a great example video of the power of bare words, please reach out if you want me to share it with you.